JSDemos/Demos/Scheduler/Editing/description.md

```diff
@@ -1 +1,13 @@
-This example demonstrates appointment editing. You can add, update and delete appointments, as&nbsp;well as&nbsp;modify them by&nbsp;dragging or&nbsp;resizing. Toggle the check boxes under the Scheduler to&nbsp;define which editing features are enabled.
\ No newline at end of file
+Scheduler allows users to add, update, resize, drag, and delete appointments. To control these operations, specify properties in the [editing](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/) object. In this demo, you can use the checkboxes below the Scheduler to toggle the edit operations.
```

Comment:
What do you mean by "control these operations"? What are you controlling?

Reply:
1) Controlling whether they are disabled or enabled (enabled by default).
2)
A user can specify which operations to enable:
![image](https://user-images.githubusercontent.com/66619750/119780794-e89dd700-bed2-11eb-8eb8-199d396d2f07.png)

Reply:
Thank you for the clarification.